### DISCUSSION TITLE
Raycast + Sonos

### DISCUSSION THREAD

**Discussion by AntonNiklasson** [svrooij/node-sonos-ts#186] [Show and tell]
Hi! I'm currently working on a [Raycast](https://www.raycast.com) integration, using this library. Here's the [pull request](https://github.com/raycast/extensions/pull/10041).

I'm a bit short on time right now, but I would love to help out any way I can here.

**Thank you** for all the hard work you've put in here 🥇

**Comment by svrooij** [comment]
Messages like these are exactly what the **Show and tell** section needs!
